src/test/bench_test/config.cpp

```diff
@@ -33,6 +33,7 @@ DSN_DEFINE_int32(pegasus.benchmark, threads, 1, "Number of concurrent threads to
 DSN_DEFINE_int32(pegasus.benchmark, hashkey_size, 16, "size of each hashkey");
 DSN_DEFINE_int32(pegasus.benchmark, sortkey_size, 16, "size of each sortkey");
 DSN_DEFINE_int32(pegasus.benchmark, value_size, 100, "Size of each value");
+DSN_DEFINE_int32(pegasus.benchmark, multi_count, 100, "Values count of the same hashkey");
```

Comment:
Could you plz move all the configs (all DSN_DEFINE_*) to file `src/test/bench_test/benchmark.cpp`, where is closer to the place use them?